provider/bidengine/pricing_test.go

```diff
@@ -545,6 +546,20 @@ func Test_ScriptPricingWritesJsonToStdin(t *testing.T) {
 	}
 }
 
+func Test_ScriptPricingFromScript(t *testing.T) {
+	scriptPath, err := filepath.Abs("testdata/usd_pricing_oracle.sh")
```

Comment:
Wouldn't this actually wind up making the unit test dependent on coin gecko (or whatever pricing oracle)? I don't think that is a good idea.

Reply:
Done. Added a mock HTTP server.